presto-main/src/main/java/io/prestosql/sql/analyzer/StatementAnalyzer.java

```diff
@@ -925,6 +924,7 @@ protected Scope visitCreateMaterializedView(CreateMaterializedView node, Optiona
             Scope queryScope = analyzer.analyze(node.getQuery(), scope);
 
             accessControl.checkCanCreateView(session.toSecurityContext(), viewName);
+            accessControl.checkCanCreateTable(session.toSecurityContext(), viewName);
```

Comment:
@martint I had proposed that we perform `checkCanCreateTable` access check on storage table, but storage table name is generated by the connector and is not available at this point in code. 
To keep code symmetric, dropMaterializedView also performs the `checkCanDropTable` check on materialized view.

Reply:
Currently syntax does not allow the user to specify the storage table name (even though the extended syntax has the provision to build it in the future). So, I will remove the `accessControl.checkCanCreateTable` and `accessControl.checkCanDropTable` checks.